CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
 - Fallback to user's log path if the default log path (`$PIPX_HOME/logs`) is not writable to aid with pipx being used for multi-user (e.g. system-wide) installs of applications
 - Fix wrong interpreter usage when injecting local pip-installable dependencies into venvs
 - add pre-commit hook support
-- Don't show escaped backslashes for paths in console output 
+- Don't show escaped backslashes for paths in console output
+- [docs] Add more examples for `pip run`
```

Comment:
Should this be `pipx run`?

Reply:
Whoops! I've just added a new commit to fix it.